docs/source/quick-start.rst

```diff
@@ -347,3 +347,140 @@ with out-of-band message processing:
+    >>> exc_crud.err
+    <tarantool.crud.CrudError object at 0x10a276950>
+    >>> exc_crud.err. # fields like in Lua implementation.
+    exc_crud.err.class_name  exc_crud.err.err         exc_crud.err.file        exc_crud.err.line        exc_crud.err.str         
+    >>> exc_crud.err.class_name
+    'InsertError'
+    >>> exc_crud.err.str
+    'InsertError: Failed to insert: Duplicate key exists in unique index "primary_index" in space "tester" with old tuple - [3500, 300, "Rob"] and new tuple - [3500, 300, "Rob"]'
+    >>> exc_crud.res is None # using only in case of *_many.
+    True
+
+    # In case of batch operation (*_many), CrudModuleError contains both result and errors:
+    >>> try:
+    ...     res = conn.crud_insert_object_many('tester', ({'id':1800,'bucket_id':100,'name':'Mike'}, {'id':2800,'bucket_id':100,'second_name':'Bill'}, {'id':3800,'bucket_id':100,'name':'Bob'}, {'id':1,'bucket_id':100,'name':'Rob'}), {'timeout':100, 'rollback_on_error':False})
+    ... except CrudModuleError as e:
+    ...     exc_crud = e
+    ... 
+    >>> exc_crud
+    CrudModuleError(<tarantool.crud.CrudResult object at 0x11a56f310>, [<tarantool.crud.CrudError object at 0x11a56e9e0>, <tarantool.crud.CrudError object at 0x11a56f490>])
+    >>> exc_crud.
+    exc_crud.args             exc_crud.errs             exc_crud.res              exc_crud.with_traceback(  
+    >>> exc_crud.res # some of the rows were inserted.
+    <tarantool.crud.CrudResult object at 0x11a56f310>
+    >>> exc_crud.res.rows
+    [[1800, 100, 'Mike'], [3800, 100, 'Bob']]
+    >>> exc_crud.errs # some of the rows were not inserted.
+    [<tarantool.crud.CrudError object at 0x11a56e9e0>, <tarantool.crud.CrudError object at 0x11a56f490>]
+    >>> exc_crud.errs[0].str
+    'CallError: Failed for 56d9c861-a8ac-459c-9715-09094ebb94a7: Function returned an error: Duplicate key exists in unique index "primary_index" in space "tester" with old tuple - [1, 100, "Mike"] and new tuple - [1, 100, "Rob"]'
+    >>> exc_crud.errs[1].str
+    'InsertManyError: Failed to flatten object: FlattenError: Object is specified in bad format: FlattenError: Unknown field "second_name" is specified'
+    >>> exc_crud.args
+    (<tarantool.crud.CrudResult object at 0x11a56f310>, [<tarantool.crud.CrudError object at 0x11a56e9e0>, <tarantool.crud.CrudError object at 0x11a56f490>])
+
+If module crud not found on the router:
+
+.. code-block:: python
+
+    >>> try:
+    ...     res = conn.crud_insert('tester', (22221,300,'Rob'))
+    ... except CrudModuleError as e:
+    ...     exc_crud = e
+    ... except DatabaseError as e:
+    ...     exc_db = e
+    ... 
+    >>> exc_db
+    DatabaseError(33, "Procedure 'crud.insert' is not defined. Ensure that you're calling crud.router and user have enough grants")
+
+Using :meth:`~tarantool.Connection.crud_select` and :meth:`~tarantool.Connection.crud_unflatten_rows`:
+
+.. code-block:: python
+
+    >>> try:
```

Comment:
Again, I don't think that it is the Python way to wrap any function call in try/catch. Let's not write in such a way until you've want to work with exception as an object.

Reply:
Done